emscripten.py

```diff
@@ -2276,17 +2265,19 @@ def main(args, compiler_engine, cache, temp_files, DEBUG):
              temp_files=temp_files, DEBUG=DEBUG)
 
 
-def _main(args=None):
-  if args is None:
-    args = sys.argv[1:]
+def _main():
+  if os.environ.get('EMCC_FAST_COMPILER') == '0':
```

Comment:
actually we can just remove these 3 lines about `EMCC_FAST_COMPILER` - that's warning about something we removed 3 years ago or more, at this point.

Reply:
Done

Reply:
thanks, lgtm too